### DISCUSSION TITLE
Question/问题

### DISCUSSION THREAD

**Discussion by Xiuyuan121** [immortalwrt/immortalwrt#857] [Q&A]
我目前使用了小米AX6S路由器，刷入了ImmortalWRT18.06KNAPSHOT版，但是我在OpenWRT Project项目官网下载到了适用于小米AX6S的固件，请问我是否可以通过LuCi的备份/升级功能升级至官方包或其他改版？

**Comment by bradfordzhang** [comment]
可以 但是不能保留配置文件

**Comment by 1715173329** [comment]
可以利用 luci 刷写，但不能保存配置文件
